README.md

```diff
@@ -0,0 +1,198 @@
+# Wagtail Content Import
+
+A module for Wagtail that provides functionality for importing page content from third-party sources.
+Currently supports importing Google Docs into a StreamField.
+
+## Getting Started
+
+### Requirements:
+* Django 2.2
+* Wagtail 2.2
+* Python 3
+
+### To install:
+ 1. Run `python3 pip install wagtail-content-import`.
+ 2. Add `'wagtail_content_import'` and `'wagtail_content_import.pickers.google'` to `INSTALLED_APPS`.
+ 
+### To set up:
+ Wagtail Google Docs integration relies on Google APIs, which you will first need to enable for your project:
+
+1. Navigate to the [Google API Library](https://console.developers.google.com/apis/library). Select a project for your Wagtail site, or create a new one now.
+
+2. Find and enable the [Google Docs](https://console.developers.google.com/apis/library/docs.googleapis.com) and [Google Drive](https://console.developers.google.com/apis/library/drive.googleapis.com) APIs.
+    
+3. Find and enable the [Google Picker](https://console.developers.google.com/apis/api/picker.googleapis.com) API, and copy its API key to the setting `GOOGLE_PICKER_API_KEY`.
+
+4. Open the [Credentials](https://console.developers.google.com/apis/credentials) page in the API Console.
+
+5. Select `Create credentials`, then `OAuth client ID`
+
+6. If you haven't already configured the consent screen, you will need to configure this now.
+
+    1. Under `Scopes for Google APIs`, click `Add scope`.
+
+    2. Add `../auth/documents.readonly` and `..auth/drive.readonly` scopes.
```

Comment:
Missing a slash before `auth` in `..auth/drive.readonly`?

Reply:
Thanks, fixed!